docs/sources/auth/saml.md

```diff
@@ -0,0 +1,136 @@
++++
+title = "SAML Authentication"
+description = "Grafana SAML Authentication"
+keywords = ["grafana", "saml", "documentation", "saml-auth"]
+aliases = ["/auth/saml/"]
+type = "docs"
+[menu.docs]
+name = "SAML"
+parent = "authentication"
+weight = 5
++++
+
+# SAML Authentication
```

Comment:
It seems it's better to use one top level heading

Reply:
I have changed this to use only one h1 and make every other an h2. However, the error from the link does not go away. I'm guessing it does not play well with the top-level commands of Hugo 😄

Reply:
Heh, yeah I'm seeing that too, apparently that's a false positive -https://github.com/markdownlint/markdownlint/issues/176, which is triggered by our code examples :((